tests/vxlan/vnet_utils.py

```diff
@@ -0,0 +1,122 @@
+import json
+import logging
+import vnet_config as vc
+import yaml
+
+from jinja2 import Template
+from os import path
+from time import sleep
+
+logger = logging.getLogger(__name__)
+
+def gen_vnet_config(mg_facts, num_vnet, num_routes, num_endpoints):
+    """
+    @summary: Generates and stores the VNET configuration
+    @param mg_facts: Minigraph facts
+    @param num_vnet: Number of VNETs
+    @param num_routes: Number of routes
+    @param num_endpoints: Number of endpoints
+    """
+    logger.info("Generating VNet configuration")
+    vc.VNET_CONFIG = yaml.safe_load(Template(open("templates/vnet_config.j2").read())
+                                .render(mg_facts, ipv6_vxlan_test=vc.IPV6_VXLAN_TEST, 
+                                        num_vnet=num_vnet, num_routes=num_routes, num_endpoints=num_endpoints))
+
+def render_template_to_host(template_name, host, dest_file, *template_args, **template_kwargs):
+    """
+    @summary: Renders a template with the given arguments and copies it to the host
+    @param template_name: A template inside the "templates" folder (without the preceding "templates/")
+    @param host: The host device to copy the rendered template to
+    @param dest_file: The location on the host to copy the rendered template to
+    @param *template_args: Any arguments to be passed to j2 during rendering
+    @param **template_kwargs: Any keyword arguments to be passed to j2 during rendering
+    """
+    # Combine all dictionaries given in template_args
+    template_args_iter = iter(template_args)
+    try:
+        combined_template_args = next(template_args_iter).copy()
+        for arg in template_args_iter:
+            combined_template_args.update(arg)
+    except StopIteration:
+        combined_template_args = {}
```

Comment:
Is there a reason this doesn't work here:
```suggestion
    combined_template_args = []
    for arg in template_args:
        combined_template_args.append(arg)
```

Reply:
`Template.render()` requires a `dict` or subclass